meinberlin/templates/a4reports/emails/report_moderators.en.email

```diff
@@ -18,7 +18,7 @@
 in the project you moderate, a contribution was reported as inappropriate.
 
 Reported contribution: <i style="color: #818181;">"{{ report.content_object }}"</i>,
-created by {{ report.content_object.creator.username }} on {% language 'de' %} {{ report.content_object.created }} {% endlanguage %}
+created by {{ report.content_object.creator.username }} on {% language 'en' %} {{ report.content_object.created }}|date:"F d Y, H:i"}} o'clock.{% endlanguage %}
```

Comment:
In the English emails, I think it's a good idea to keep the language en tag and remove the specified format, as this would read very weird in English, I think.

Reply:
In the English ones, I still think, we should remove it. :)

Reply:
But I adapted the filter to the English format here :) Ok, maybe the time would look weird.. will remove it!

Reply:
Haha, so it was already strange before. :)